packages/analytics-browser/test/plugins/diagnostic.test.ts

```diff
@@ -0,0 +1,15 @@
+import { Diagnostic } from '../../src/plugins/diagnostic';
+
+describe('Diagnostic', () => {
```

Comment:
We should also add tests for the client. e.g.
1. Should track diagnostics on 429 error
2. Should track diagnostics on 500 error
3. ... others
4. Should not track diagnostics on 200

Reply:
Tests added in `analytics-browser/test/browser-client.test.ts`